### DISCUSSION TITLE
CURIS Summer Putnam Eval Benchmark, SNAP Setup Discussion

### DISCUSSION THREAD

**Discussion by eric-h-chen** [brando90/snap-cluster-setup#6] [General]
Hey, I've finished the SNAP setup up to setting up the python project. I'll work on more today.

**Comment by emxia18** [comment]
some issues eric and i found while running through the tutorial and their fixes
1. Issues with torch versions, this gave errors in the test_vllm portion
Fix: we made a new conda environment with python version 3.9 and then did the following pip installs:
pip install torch==2.2.1
pip install transformers vllm==0.4.1 xformers==0.0.25 and proceeded to clone snap-cluster-setup github into this new environment. 
2. Many packages needed to be installed through the process
Fix: rather simple, just import what the error message said (pip instal ___ )
3. Some of the code was incorrect
simple_train .py: Line 12: could not import “​​get_ai4m_v0, get_data_set_args, load_dataset_block_size” from utils --> simply remove this import, all you need is import utils eval_hf
simple_train.py: Line 160: eval_dataset is not a valid parameter to the eval_hf function --> simply remove
